src/context.js

```diff
@@ -165,7 +166,8 @@ export default class Context {
 	}
 
 	getAttribute(attrName) {
-		return this._attrs.find((attr) => attr.name === attrName).value;
+		const reversedAttrs = [...this._attrs].reverse();
+		return reversedAttrs.find((attr) => attr.name === attrName).value;
```

Comment:
I think Array.find is not supported in IE10 - can you verify that?
It's also not very efficient to create a new array just to find an element.